src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/polyline-visualizer.ts

```diff
@@ -107,23 +116,133 @@ export class FootprintPolylineVisualizer extends BaseFootprintsVisualizer {
 		return { isFavorites, isActive, isDisplayed };
 	}
 
+	private getFeatureType(feature: Feature): 'MultiLineString' | 'Point' | 'LineString' | 'MultiPolygon' {
+		return feature && feature.getGeometry().getType();
```

Comment:
add console warn for not supported types
enforce supported footprint type (with explanation remark) on the providers
check that all providers supply valid types (polygon -> multi-polygon)